lighthouse-core/runner.js

```diff
@@ -258,6 +258,8 @@ class Runner {
       const normalizedGatherSettings = Object.assign({}, artifacts.settings, overrides);
       const normalizedAuditSettings = Object.assign({}, settings, overrides);
 
+      // require('assert').deepStrictEqual(normalizedGatherSettings, normalizedAuditSettings);
```

Comment:
what if...we just replaced this with a warning instead :D

I don't think this has ever once, a single time caught the thing it was trying to prevent and is 100% annoying

Reply:
to be fair, in some of these cases, it meant I hadn't updated the `artifacts.json` fixture. I just needed a diff to figure out what to do.

but yes most of the time i hate this check and comment out the original isDeepEqual instead.